tests/Unit/Helpers/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -500,22 +496,79 @@ struct VectorOrArrayAt {
 // given an explicit template parameter pack `Wraps`, wrap the elements of
 // `operand`, passed by pointer, element-by-element, and return the
 // resulting tuple of wrapped elements.
-template <typename... Wraps, typename... Operands, size_t... Is>
+template <typename WrapTuple, typename... Operands, size_t... Is>
 auto wrap_tuple(std::tuple<Operands...>& operand_values,
+                const WrapTuple& wraps,
                 std::index_sequence<Is...> /*meta*/) noexcept {
-  return std::make_tuple(wrap(Wraps{}, get<Is>(operand_values))...);
+  return std::make_tuple(wrap(get<Is>(wraps), get<Is>(operand_values))...);
 }
 
+// CHECK forwarding for parameter pack expansion. Return value also required for
+// easy parameter pack use.
+template <typename Approx>
```

Comment:
Since `Approx` is an actual type in Catch, I'm a but surprised that it is a template parameter.  You already include Catch. You will need to mark the function `inline`. If you prefer `FORCE_INLINE` add the include for that